### ISSUE TITLE
The problem of incorrect transcoding in Chinese

### ISSUE THREAD

**Issue by zhbgs2000** [open-source-parsers/jsoncpp#803]
```
	Json::Value json_temp; 
	json_temp["name"]	= "中国234";
	string name			= json_temp["name"].asString();
	Json::Value root;  		
	root["key_number"]	= Json::Value(12345);					
	root["key_boolean"] = Json::Value(false);					
	root["key_double"]	= Json::Value(12.345);					
	root["key_object"]	= json_temp;													
						
	Json::StreamWriterBuilder builder;
	builder["commentStyle"] = "None";
	builder["indentation"] = "";
	//string s		= fast_writer.write(root);
	//string_t s	= utility::conversions::to_string_t(fast_writer.write(root));
	//string result = fast_writer.write(root);
	string result = Json::writeString(builder, root);
	cout <<"name: "<< name << endl;	//is ok
	cout <<"result: "<< result << endl; //no ok
       ......
```

print info:
name: 中国234
result: {"key_boolean":false,"key_double":12.345000000000001,"key_number":12345,"key_object":{"name":"\u0590\u067a234"}}

jsoncpp version:1.8.4
ide: visual studio 2017 c++
The value of the variable name is displayed in Chinese, and the Chinese contained in variable result cannot be displayed properly.

**Comment by zhbgs2000** [comment]
```
Json::Value json_temp;
json_temp["name"]	= "中国234";
string name	= json_temp["name"].asString();
Json::Value root;
root["key_number"]	= Json::Value(12345);
root["key_boolean"] = Json::Value(false);
root["key_double"]	= Json::Value(12.345);
root["key_object"]	= json_temp;

Json::StreamWriterBuilder builder;
builder["commentStyle"] = "None";
builder["indentation"] = "";
//string s		= fast_writer.write(root);
//string_t s	= utility::conversions::to_string_t(fast_writer.write(root));
//string result = fast_writer.write(root);
string result = Json::writeString(builder, root);
cout <<"name: "<< name << endl;	//is ok
cout <<"result: "<< result << endl; //no ok
   ......
```

print info:
name: 中国234
result: {"key_boolean":false,"key_double":12.345000000000001,"key_number":12345,"key_object":{"name":"\u0590\u067a234"}}

The value of the variable name is displayed in Chinese, and the Chinese contained in variable result cannot be displayed properly.

**Comment by robot0x** [comment]
it seems writeString function transform chinese character to unicode after see source code.

**Comment by zhbgs2000** [comment]
use jsoncpp version 1.8.3 is ok

**Comment by cdunn2001** [comment]
See #687. We need someone to add a configuration setting to determine whether to assume the internal representation is UTF-8.

**Comment by maxguru** [comment]
Is there any progress on this? As far as I can tell there is no way to tell jsoncpp to not escape UTF-8 characters with \u with the post 1.8.3 versions. It would be really nice to have this feature.

**Comment by dota17** [comment]
You can try the new feature - `emitUTF8`, which is merged from #1045 .

**Comment by maxguru** [comment]
@dota17 nice, thanks!
